macs/mac-exec

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+HOSTS=(
+	"hetzner@enormous-catfish.mac.nixos.org"
+	"hetzner@growing-jennet.mac.nixos.org"
+	"hetzner@intense-heron.mac.nixos.org"
+	"hetzner@maximum-snail.mac.nixos.org"
+	"hetzner@sweeping-filly.mac.nixos.org"
+	"customer@eager-heisenberg.mac.nixos.org"
+	"customer@kind-lumiere.mac.nixos.org"
+)
+
+for host in "${HOSTS[@]}"; do
+	# shellcheck disable=SC2068
+	(ssh "${host}" -- $@ 2>&1| sed -e "s/^/${host} | /") &
+done
```

Comment:
```
PIDS=()

for host in "${HOSTS[@]}"; do
    # shellcheck disable=SC2068
    (ssh "${host}" -- $@ 2>&1 | sed -e "s/^/${host} | /") &
    PIDS+=($!)
done

for pid in "${PIDS[@]}"; do
    wait $pid
done
```

Reply:
Maybe something like `wait "${PIDS[@]}"` at the end.

Reply:
Applied in 63ea54bff9b83bea9a2307006defe17402da9c76